e2e/test/scenarios/onboarding/search/search.cy.spec.js

```diff
@@ -63,6 +70,28 @@ describe("scenarios > auth > search", () => {
   });
 });
 
+describeWithSnowplow.only("scenarios > search", () => {
```

Comment:
Maybe need to remove this

```suggestion
describeWithSnowplow("scenarios > search", () => {
```

Reply:
Fixed